### ISSUE TITLE
strange alternating sound with laser guitar pickup

### ISSUE THREAD

**Issue by tzenn** [alex6679/teensy-4-usbAudio#4]
Hey! I tried to reach out on the forum but its been a week and I'm still waiting for admin approval to get in so I'll just post it here for now. 

Video footage of the issue: https://youtu.be/x1HCdltNRwA 

**Environment:**
- Teensy 4.1
- Using modified core files from alex6679/teensy-4-usbAudio
- FL Studio with ASIO4ALL driver
- 6-channel setup (showing as 6 mono / 3 stereo channels in FL Studio)

**Hardware Setup:**
- 6 photodiodes with 100kΩ load resistors between analog pins (A0-A5) and ground
- Each photodiode is part of a laser guitar string pickup system where laser beams are interrupted by guitar strings
- Photodiodes show correct response in serial monitor:
  - 8-14 range with no laser
  - Up to ~150 when laser hits the photodiode

**Code Being Used:**
```cpp
#include <Audio.h>
#include <Wire.h>
#include <SPI.h>
#include <SD.h>
#include <SerialFlash.h>

const int STRING_COUNT = 6;
const int photodiodePins[STRING_COUNT] = {A0, A1, A2, A3, A4, A5};

AudioInputAnalog         adc1(photodiodePins[0]);  // String 1 (high E)
AudioInputAnalog         adc2(photodiodePins[1]);  // String 2 (B)
AudioInputAnalog         adc3(photodiodePins[2]);  // String 3 (G)
AudioInputAnalog         adc4(photodiodePins[3]);  // String 4 (D)
AudioInputAnalog         adc5(photodiodePins[4]);  // String 5 (A)
AudioInputAnalog         adc6(photodiodePins[5]);  // String 6 (low E)

AudioOutputUSB           usb1;

AudioConnection          patchCord1(adc1, 0, usb1, 0);
AudioConnection          patchCord2(adc2, 0, usb1, 1);
AudioConnection          patchCord3(adc3, 0, usb1, 2);
AudioConnection          patchCord4(adc4, 0, usb1, 3);
AudioConnection          patchCord5(adc5, 0, usb1, 4);
AudioConnection          patchCord6(adc6, 0, usb1, 5);

void setup() {
  AudioMemory(24);
  Serial.begin(115200);
}

void loop() {
  for(int i = 0; i < STRING_COUNT; i++) {
    int rawAmplitude = analogRead(photodiodePins[i]);
    Serial.print("String ");
    Serial.print(i + 1);
    Serial.print(": ");
    Serial.println(rawAmplitude);
  }
  delay(10);
}
```

**Issue Description:**
1. First analog input (A0) outputs no sound at all
2. The remaining channels (A1-A5) output a strange buzzing noise that alternates between channels
3. This occurs despite the serial monitor showing correct photodiode response to laser interruption
4. The alternating buzzing noise persists regardless of laser/photodiode state

**Additional Context:**
- The photodiodes are confirmed working through serial monitor output
- ASIO4ALL is properly detecting all 6 channels
- FL Studio is receiving the channels as expected (6 mono / 3 stereo)
- The issue appears to be specifically with the audio output, not the sensor readings

**Questions:**
1. Is there a known issue with the first channel (A0) in multi-channel configurations?
2. What might cause the alternating buzzing between channels despite correct sensor readings?
3. Are there any specific initialization requirements for multi-channel audio that might be missing?

Thanks for your time!

**Comment by h4yn0nnym0u5e** [comment]
Fairly sure the Audio library doesn't support multiple instances of `AudioInputAnalog` - if you try to place two in the Design Tool, you get the yellow warning triangle signalling a hardware clash.

**Comment by alex6679** [comment]
Does the 'multiple AudioInputAnalog instance' issue explain your problem?

**Comment by tzenn** [comment]
hmmm I'm not sure. I haven't used the design tool but I'll give it a shot. Do you know of any other ways it can be done?

**Comment by h4yn0nnym0u5e** [comment]
Not really. The official audio library only supports stereo input using the built-in ADC for the (obsolete) Teensy 3.x, and the code for the mono Teensy 4.x is riddled with commented-out code and TODO comments. It might be (probably is) possible for someone versed in the ADC hardware to make modifications to allow multiple channels, but I've no idea how many channels would be achievable.
